test/sql/table_function/sqlite_master.test

```diff
@@ -36,7 +38,7 @@ create table tconstraint1(i integer primary key default(3), j blob not null);
 query IIIII
 SELECT * FROM sqlite_master WHERE name='tconstraint1';
 ----
-table	tconstraint1	tconstraint1	0	CREATE TABLE tconstraint1(i INTEGER PRIMARY KEY DEFAULT(3), j BLOB NOT NULL);
+table	tconstraint1	tconstraint1	0	CREATE TABLE tconstraint1(i INTEGER DEFAULT(3) PRIMARY KEY, j BLOB NOT NULL);
```

Comment:
Can we fix the SQLite tests by having the same catalog clear in `duckdb_tables`, i.e. instead of doing `table.ToSQL()` we do `table.GetInfo()` followed by `catalog.clear()` and `info.ToSQL()`